### ISSUE TITLE
Update tracee-rules UX to align with tracee-ebpf

### ISSUE THREAD

**Issue by grantseltzer** [aquasecurity/tracee#499]
I think we should align with tracee-ebpf's UX. 
perhaps something like:

```
--input-tracee file:/path/to/file
--input-tracee option:{eot}
--input-tracee format:gob/json 
```

stdin can be given as a file (like in the readme) or we can create an alias for it

_Originally posted by @itaysk in https://github.com/aquasecurity/tracee/discussions/495#discussioncomment-330123_

**Comment by grantseltzer** [comment]
@itaysk What did you have in mind for when a user specifies `--input-tracee option:eot`?

**Comment by itaysk** [comment]
take a loot at the same option in tracee-ebpf, if it's still not clear let's have a chat and I'll explain

**Comment by yanivagman** [comment]
I wrote this comment in the PR, but maybe its better that we also discuss this here:
To be future proof, I think we should change this flag name to be simply `input`.
Then, we can have something like:
`--input source:key=value`
WDYT?

**Comment by itaysk** [comment]
closed by #500
